internal/preferences.go

```diff
@@ -180,5 +188,6 @@ func (p *InMemoryPreferences) RemoveValue(key string) {
 func NewInMemoryPreferences() *InMemoryPreferences {
 	p := &InMemoryPreferences{}
 	p.values = make(map[string]interface{})
+	p.waitGroup = &sync.WaitGroup{}
 	return p
```

Comment:
```go
 	return &InMemoryPreferences{
		values: make(map[string]interface{}),
		waitGroup: &sync.WaitGroup{},
	}
```
I think this would be a bit cleaner while you're at it.

Reply:
sure

Reply:
It would be cleaner if you return it directly instead of storing it to a variable first.

Reply:
Goimports also complains about the formatting.

Reply:
updated